tests/scene/test_camera_3d.h

```diff
@@ -115,6 +115,79 @@ TEST_CASE("[SceneTree][Camera3D] Getters and setters") {
 		CHECK(test_camera->get_size() == size);
 	}
 
+	SUBCASE("Frustum planes") {
+		constexpr real_t sqrt2_half = 1.41421356237 / 2;
+
+		constexpr real_t fov = 90.0, size = 6.0;
+		constexpr real_t near1 = 0.1, near2 = 0.5;
+		constexpr real_t far1 = 1001.0, far2 = 1005.0;
+
+		SubViewport *mock_viewport = memnew(SubViewport);
+
+		mock_viewport->set_size(Vector2i(512, 512)); // aspect 1
+		SceneTree::get_singleton()->get_root()->add_child(mock_viewport);
+		mock_viewport->add_child(test_camera);
+		test_camera->set_keep_aspect_mode(Camera3D::KeepAspect::KEEP_HEIGHT); // don't flip fov
```

Comment:
IIRC, this is the default, so you can remove it.

Reply:
I'm trying to figure out why some automated tests do not pass.
I saw this snippet is used at different places in this file and took a stab at it.
Bad luck, it still doesn't pass...

Reply:
Any idea welcome, I'm kinda stuck ...